install_requirements.sh

```diff
@@ -15,12 +15,13 @@ sudo apt -y install \
    ffmpeg \
    iproute2 \
    python3-pip \
+   python3-pexpect \
```

Comment:
please add policykit-1 here for pkexec and move installing pexpect for python into the `pipenv install` call

Reply:
Done.